### ISSUE TITLE
Trending Lists

### ISSUE THREAD

**Issue by mattyg** [GeekGene/MewsFeed#72]
Clone & extend the concept from twitter of Trending lists.

**Overview**
- A list is comprised only of Agents, Mews, Replies, Hashtags, Cashtags, or some combination of content types
- Each list is *global* for the entire DHT
- It is too expensive for every user to generate their own list via a DHT query, thus there will need to be a class of users who act as "indexing" service providers: generating the trending lists which other users simply consume.

**Comment by mattyg** [comment]
I feel a bit conflicted about this, personally.

On the one hand, it would be really cool to flex what we can do in holochain -- showing that large scale indexing of content is possible. Maybe its a good thing to have *some* content that is globally visible to every user, and is consistent for all users, for some kind of coherence.

On the other hand, maybe it's not so great to have globally visible content for all users. I'm almost never interested in trending topics on twitter, so to me it's just attention pollution. Why do we need to have any kind of global coherence? I seems to me that that scale of coherence needs to be built up from individuals -> groups -> larger groups -> .... -> global, to be authentic in any way.

**Comment by dauphin3** [comment]
I agree about the confliction.. I think this is a perfect use case for TrustGraph so that way an Agent can get trending feeds based on their network rollup depth and other settings such as confidence in a topic for example or metrics of activity of who/what theyve interacted with

**Comment by bierlingm** [comment]
Agreed on this with respect to TrustGraph being a great fit here. #94 

I guess there are two things coming together in the Twitter concept of trending lists.

1. "I want to see what _everyone else_ (globally) is talking about"
2. "I want to see what _people I find interesting (i.e. follow)_ are talking about"

I think in MewsFeed, we can think about 1. as belonging to the bucket of discovery which we haven't built much for yet. Something that's easy to do in a centralized system but harder in Holochain-land, and perhaps not even desirable according to the vision we have. So I think we can still serve the underlying desire by providing discovery tools, but I'm not sure we want to mimic the global nature of Twitter there.

Serving the desire of 2. would be pretty easy to do with just Holochain tools, but even easier with Trustgraph in the mix.

A) Check out people that the people you follow follow
B) Check out the user(s) that multiple people you follow already follow
C) Check out this mew that multiple people you follow li(c)ked
D) Check out this hashtag (term, cashtag, reply, etc) that has been engaged with a bunch
etc
